crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -559,13 +559,13 @@ impl PendingTransactionsReceiver {
 
 /// A structure to manage and provide access to a stream of full transaction details.
 #[derive(Debug, Clone)]
-struct FullTransactionsReceiver<T: PoolTransaction> {
+struct FullTransactionsReceiver<T: PoolTransaction<Consensus = TransactionSignedEcRecovered>> {
```

Comment:
This type needs `Consensus` bound because it performs an actual `into_consensus` conversion when constructing `FilterChanges`. However, its underlying types, including `NewSubpoolTransactionStream`, `NewTransactionEvent` and `ValidPoolTransaction` doesn't seem to need it. Am I missing something here?